Library/Formula/twoping.rb

```diff
@@ -1,16 +1,11 @@
 class Twoping < Formula
   desc "Ping utility to determine directional packet loss"
   homepage "http://www.finnie.org/software/2ping/"
-  url "http://www.finnie.org/software/2ping/2ping-3.0.1.tar.gz"
-  sha256 "d6997cd1680151e6f7d5e60137d45cd41bf385d26029878afdaaf5dc4f63dcc4"
-  revision 1
+  url "http://www.finnie.org/software/2ping/2ping-3.1.0.tar.gz"
+  sha256 "0f5b4e10f89dd99ef04e2cc0564d6673ceae897ed0bf59b636553ec6aa1c7a92"
+  head "https://github.com/rfinnie/2ping.git"
 
-  bottle do
-    cellar :any_skip_relocation
-    sha256 "f63e7af668f34e6b293e5a554832a7432de14cbb71c15e0a225c6ea1ce3e3924" => :el_capitan
-    sha256 "13794ca27a86eec6ede9a1f1cd3d01cda5f675db91c2e937471bf7d718c0fed4" => :yosemite
-    sha256 "b2a81bf692ca4fc69367536e069634438dd2dce53a9f0f7e05b6749342a96db4" => :mavericks
-  end
+  bottle :unneeded
```

Comment:
You still need bottles here; there's only no bottles when there's no compile stage, and there's a compile stage here.